modules/farm/spec/01_state.md

```diff
@@ -0,0 +1,61 @@
+<!--
+order: 1
+-->
+
+# State
+
+## Params
+
+`Params` defines the parameters that the farm module can manage through the `gov` module.
+
+```go
+type Params struct {
+    CreatePoolFee      sdk.Coin 
+    MaxRewardCategoryN uint32                                  
```

Comment:
I think `MaxRewardCategories` is clear enough.

Reply:
will update in next PR